### PR TITLE
Add k3s-cis support for permissive and hardened profiles 

### PR DESCRIPTION
**Pull request by MonzElmasry**
Note: this pr is missing the security scans image version update TBDL 
issue : https://github.com/k3s-io/k3s/issues/1504

### CODE REVIEW

**Review by briandowns — DISMISSED** [review]
LGTM

**Review by galal-hussein — APPROVED** [review]

**Review by dweomer — DISMISSED** [review]

File: packages/rancher-cis-benchmark/charts/templates/configmap.yaml [hunk 1]
```diff
@@ -13,4 +13,5 @@ data:
     >=1.20.5: rke2-cis-1.6-profile-permissive
   eks: "eks-profile"
   gke: "gke-profile"
+  k3s: "k3s-cis-1.6-profile-permissive"
```
Comment (prachidamle) [inline packages/rancher-cis-benchmark/charts/templates/configmap.yaml:16]: is there any variation based on k8s versions? Or for all versions "k3s-cis-1.6-profile-permissive" is the default?

**Comment by prachidamle** [comment]
As noted in the description, the chart update should have the new security-scan image tag to backup these profiles.

File: packages/rancher-cis-benchmark/charts/templates/configmap.yaml (hunk 1 above)
Comment (briandowns) [reply packages/rancher-cis-benchmark/charts/templates/configmap.yaml:16]: `k3s-cis-1.6-profile-permissive` will be the default.

**Review by prachidamle — DISMISSED** [review]
LGTM

**Review by brandond — APPROVED** [review]

**Pull request merged**